src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,64 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
     public static void main(String[] args) {
-        // test your code here
+        Injector injector = Injector.getInstance("mate.jdbc");
+
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+        manufacturerService.create(new Manufacturer("Datsun", "Japan"));
+        manufacturerService.create(new Manufacturer("Toyota", "Japan"));
+
+        System.out.println("All manufacturers");
+        manufacturerService.getAll().forEach(System.out::println);
+        System.out.println(System.lineSeparator());
+
+        Manufacturer manufacturer = manufacturerService.get(1L);
```

Comment:
Let's try to not hardcode `id` values, you can get needed `id` by using `manufacturer.getId()` method. 
Please fix all similar cases (not only for manufacturers) in this class.